tekton/pipeline_run_test.go

```diff
@@ -216,7 +219,24 @@ var _ = Describe("PipelineRun", func() {
 
 		It("can add the EC task bundle parameter to the PipelineRun", func() {
 			releasePipelineRun.WithEnterpriseContractConfigMap(enterpriseContractConfigMap)
-			Expect(releasePipelineRun.Spec.Params).Should(ContainElement(HaveField("Value.StringVal", Equal(string("test-bundle")))))
+			params := releasePipelineRun.Spec.Params
+			checked := 0
+
+			for i := range params {
```

Comment:
Can't we just check the params directly (relying on the ordering)? We do it in other tests in this file as well, so should be fine. But this is fine too.

Reply:
Considering the order doesn't matter, I'd really not rather rely on the ordering if we don't have to

Reply:
ok, fine with me

Reply:
I think it would be great to do it as Martin suggest. Having said this, the whole package will require a refactor whenever we introduce dev pipelines. Test are also a bit outdated in this package. So we can do it in a different PR.